common/changes/@microsoft.azure/openapi-validator/akhilailla-make_PatchPropertiesCorrespondToPutProperties_to_staging_only_2023-09-21-21-36.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file shouldn't be in this PR.

Reply:
Builds fail without this file

Reply:
It fails because there are changes to `azure-openapi-valiadator/autorest/package.json`. If you revert them (please do that) then it will pass.